registry/storage/driver/testsuites/testsuites.go

```diff
@@ -3,6 +3,10 @@ package testsuites
 import (
 	"bytes"
 	"crypto/sha1"
+	"fmt"
```

Comment:
@RichardScothern , sorry for late, and thanks for your checking. I will amend the import order.